etc/barman.d.in/template

```diff
@@ -6,3 +6,12 @@ streaming_conninfo = host=$host port=$port user=$user_stream dbname=$dbname
 backup_method = "postgres"
 streaming_archiver = on
 slot_name = $slot_name
+
+# at present each backup costs a couple of hundred GB so without infinite
+# disk space (and money on AWS) this can't be unbounded
+retention_policy = RECOVERY WINDOW OF 3 MONTHS
+
+# http://docs.pgbarman.org/release/2.5/#backup-features
+# > incremental backup, by reusing the last backup for a server and creating
```

Comment:
Sorry, I merged this too soon, I've just seen this in the docs:

"IMPORTANT: The reuse_backup option can’t be used with the postgres backup method at this time."

Reply:
What does that even mean? barman only backs up postgres dbs...is this something that applies to us? (great catch)